src/main/java/com/dnastack/ga4gh/dataconnect/adapter/trino/QueryCleanupManager.java

```diff
@@ -18,23 +18,24 @@
     private final Jdbi jdbi;
     private final TrinoClient client;
 
-    @Value("${app.query-cleanup-timeout-in-seconds}")
+    @Value("${app.query-cleanup.timeout-in-seconds}")
     private int queryCleanupTimeoutInSeconds;
 
+    @Value("${app.query-job-cleanup.deletion-timeout-in-days}")
+    private int queryJobCleanupDeletionTimeoutInDays;
+
     public QueryCleanupManager(Jdbi jdbi, TrinoClient client) {
         this.jdbi = jdbi;
         this.client = client;
     }
 
-    @Scheduled(cron = "${app.query-cleanup-interval}")
-    public void cleanupOldQueries() {
+    @Scheduled(cron = "${app.query-cleanup.cron-interval}")
+    public void terminateOldQueries() {
         List<QueryJob> queryJobList = jdbi.withExtension(QueryJobDao.class, dao -> {
             Instant oldQueryTimestamp = Instant.now().minusSeconds(queryCleanupTimeoutInSeconds);
             return dao.getOldQueries(oldQueryTimestamp);
         });
-        if (queryJobList.isEmpty()) {
-            log.info("No old queries present, skipping cleanup.");
```

Comment:
Was this removed because it was generating lot of unnecessary logs?

Reply:
Yeah, since the method is run every 10s this ends up getting posted a lot